CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+### 1.9.1
+
+CHANGES:
+* Fix deployment of ingressclass resource via helm on some versions of Kubernetes.
+* Update the base ubi images to 8.3.
+* Renew CA cert for egress-mtls example.
+* Add imagePullSecretName support to helm chart.
```

Comment:
What do you think about adding the deprecation notice to this and the other changelog?
```suggestion
Add `controller.serviceAccount.imagePullSecretName` parameter to the helm chart. **Note**: `controller.serviceAccount.imagePullSecrets` is deprecated and will be removed in the future `1.10.0` release. 
```

Render:
Add `controller.serviceAccount.imagePullSecretName` parameter to the helm chart. **Note**: `controller.serviceAccount.imagePullSecrets` is deprecated and will be removed in the future `1.10.0` release.